### DISCUSSION TITLE
Does DXMT support persistent shader caching?

### DISCUSSION THREAD

**Discussion by NSArtem** [3Shain/dxmt#49] [General]
Does DXVK support persistent shader caching between launches, similar to .dxvk-cache?

I'm running Wuthering Waves and DXVK + metalfx upscaling give really good performance results, but stuttering on every new shader is a bit annoying.

Besides, gonna take this post as an opportunity to give huge thanks to 3Shain for the great work! DXMT is an awesome technical feat, and knowing this is a one-man-job, it's even more impressive.

**Comment by 3Shain** [comment]
(I guess you mistyped DXMT in the content)

A persistent pipeline cache will be done at some point but not prioritized at the moment.
